src/search/FileFilters.js

```diff
@@ -337,10 +344,24 @@ define(function (require, exports, module) {
             });
         }
 
+        $nameField.bind('input', function () {
+            var remainingCharacters = FILTER_NAME_CHARACTER_MAX - $(this).val().length;
+            $remainingField.text(StringUtils.format(
+                Strings.FILTER_NAME_REMAINING,
+                remainingCharacters
+            ));
+
+            if (remainingCharacters < 0) {
```

Comment:
unless i'm at lets say 3/4 of `FILTER_NAME_CHARACTER_MAX`, $remainingField could be hidden?